code/controllers/configuration/entries/general.dm

```diff
@@ -62,6 +62,8 @@
 
 /datum/config_entry/flag/log_prayer // log prayers
 
+/datum/config_entry/flag/log_craft //log crafts
+
```

Comment:
You need to also add this to config.txt along with the other log config entries so its easy to set and unset for new servers dev builds, etc.

It can likely be safely enabled by default.

Reply:
This, but honestly it doesn't even really need to be its own log at all